src/main/java/org/broadinstitute/hellbender/utils/read/FlowBasedReadUtils.java

```diff
@@ -118,22 +141,28 @@ public static int flowSumOfBaseQualities(final GATKRead read) {
         }
     }
 
-    public static boolean isFlow(final GATKRead rec) {
+    public static boolean hasFlowTags(final GATKRead rec) {
         return rec.hasAttribute(FlowBasedRead.FLOW_MATRIX_TAG_NAME)
                 || rec.hasAttribute(FlowBasedRead.FLOW_MATRiX_OLD_TAG_KR)
                 || rec.hasAttribute(FlowBasedRead.FLOW_MATRiX_OLD_TAG_TI);
     }
 
-    public static boolean isFlow(final SAMRecord rec) {
+    public static boolean hasFlowTags(final SAMRecord rec) {
         return rec.hasAttribute(FlowBasedRead.FLOW_MATRIX_TAG_NAME)
                 || rec.hasAttribute(FlowBasedRead.FLOW_MATRiX_OLD_TAG_KR)
                 || rec.hasAttribute(FlowBasedRead.FLOW_MATRiX_OLD_TAG_TI);
 
     }
 
+    public static boolean isFlowPlatform(final SAMFileHeader hdr, final GATKRead read) {
```

Comment:
Can you just add a javadoc to this?

Reply:
And also possibly update the class javadoc to point people to this method as the definitive "is this flow based and should i trigger that code" method

Reply:
fixed